src/layouts/header/manage-assets.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+    <header class="container">
```

Comment:
use class `container-fluid` instead of `container` so the full space is used. I'm now getting margins left and right on a bigger screen.

Reply:
changed to container-fluid

Reply:
changed to container-fluid